### PR TITLE
feat(DIA-556): remove collector settings ff dead code

### PR DESCRIPTION
**Pull request by araujobarret**
This PR resolves [DIA-556] 

### Description

Removes legacy code from collector settings(before ff)

#nochangelog

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-556]: https://artsyproduct.atlassian.net/browse/DIA-556?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c0f614804ee8e3f8a7d49f0bc0574ec2dcf40f69
</p>

File: src/app/store/config/features.ts [hunk 1]
```diff
@@ -81,12 +81,6 @@ export const features = {
     readyForRelease: true,
     echoFlagKey: "AREnableNewAuctionsRailCard",
   },
-  AREnableNewCollectorSettings: {
-    description: "Enable New Collector Settings",
-    readyForRelease: true,
-    echoFlagKey: "AREnableNewCollectorSettings",
-    showInDevMenu: true,
-  },
```
Comment (MrSltun) [inline src/app/store/config/features.ts:89]: **Question**: should we retire this in echo as well?

**Review by MrSltun — APPROVED** [review]
🧹🧹🧹

**Review by egdbear — APPROVED** [review]

File: src/app/Scenes/MyProfile/MyProfileHeader.tests.tsx [hunk 2]
```diff
@@ -18,107 +17,54 @@ describe("MyProfileHeader", () => {
...
+    expect(profileImage).toBeTruthy()
+    fireEvent.press(profileImage)
+    expect(navigate).toHaveBeenCalledTimes(1)
+    expect(navigate).toHaveBeenCalledWith("/my-profile/edit", {
+      passProps: { onSuccess: expect.anything() },
     })
+  })
 
-    it("renders Collector Profile info", async () => {
+  describe("new settings screen", () => {
```
Comment (egdbear) [inline src/app/Scenes/MyProfile/MyProfileHeader.tests.tsx:32]: nitpick
```suggestion
  describe("settings screen", () => {
```

File: src/app/store/config/features.ts (hunk 1 above)
Comment (araujobarret) [reply src/app/store/config/features.ts:89]: We can't remove it from there yet(too recent) but I've just opened the PR to deprecate it from the list
https://github.com/artsy/echo/pull/690

**Pull request merged**